rest/service/dashboardTemplate.go

```diff
@@ -408,18 +408,36 @@ func getLandingPageBaseLayout(x int) []models.GridItem {
 			X:                    0,
 			Y:                    2,
 		},
+		models.GridItem{
+			BaseWidgetDimensions: WidgetMapping[models.RecentlyVisited].Defaults,
+			ID:                   "recentlyVisited#recentlyVisited",
+			X:                    3,
+			Y:                    0,
+		},
+		models.GridItem{
+			BaseWidgetDimensions: WidgetMapping[models.Quay].Defaults,
+			ID:                   "quay#quay",
+			X:                    3,
+			Y:                    0,
+		},
 		models.GridItem{
 			BaseWidgetDimensions: WidgetMapping[models.FavoriteServices].Defaults,
 			ID:                   "favoriteServices#favoriteServices",
 			X:                    0,
 			Y:                    3,
 		},
 		models.GridItem{
-			BaseWidgetDimensions: WidgetMapping[models.LearningResources].Defaults,
```

Comment:
@epwinchell I see we removed the `LearningResources` widget from the base layout - can we confirm that was intentional? I looked in the Jira ticket and PR description but didn't see anything indicating we should remove it?

Reply:
@florkbr It's intentional. Here is the document: https://docs.google.com/document/d/171N-nMMIvozIVGO0FXXvG1aBLZmPwcDFDpbz3VuNi6o/edit#heading=h.1jgo6w5l38hg